exercises/practice/error-handling/error-handling.js

```diff
@@ -0,0 +1,6 @@
+export const processString = (input) => {
+  //TODO: implement this
```

Comment:
In all other stubs we `throw new Error('...')` (I don't recall the exact error message). We probably want to use that!